### PR TITLE
chore(shopify): refactor use one map to fix test warning

### PR DESCRIPTION
**Pull request by cbernat**
@fcoronelmakingsense the issue you've mentioned earlier fixed here.

### CODE REVIEW

File: src/components/Integrations/Shopify/Shopify.js [hunk 1]
```diff
@@ -138,60 +138,64 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
...
-                                  <FormattedDate value={shop.synchronization_date} timeZone="UTC" />
-                                </strong>
-                              </p>
-                              <p>
-                                {_('shopify.header_store')} <strong> {shop.shopName} </strong>
-                              </p>
-                            </li>
-                          </ul>
+              {shopifyState.shops.map((shop) => (
+                <div key={shop.shopName}>
```
Comment (andresmoschini) [inline src/components/Integrations/Shopify/Shopify.js]: Is the key duplicated?

Comment (andresmoschini) [reply src/components/Integrations/Shopify/Shopify.js]: image

Comment (andresmoschini) [inline src/components/Integrations/Shopify/Shopify.js]: On the other hand, are these keys safe? I mean, we cannot have duplicated names?

Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: Yes I've seen them thanks.

Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: I don't think we can, they are shopify urls to the store.

Comment (andresmoschini) [reply src/components/Integrations/Shopify/Shopify.js]: 👍

**Pull request merged**